google/cloud/forseti/common/gcp_api/storage.py

```diff
@@ -419,6 +442,28 @@ def get_text_file(self, full_bucket_path):
             LOGGER.exception('Unable to download file.')
             raise
 
+    def download_file(self, full_bucket_path, output_file):
+        """Downloads a copy of a file from GCS.
+         Args:
+            full_bucket_path (str): The full path of the bucket object.
+            output_file (file): The file object to write the data to.
+         Returns:
+            int: Total size in bytes of file.
+         Raises:
+            HttpError: HttpError is raised if the call to the GCP storage API
+                fails
+        """
+        bucket, object_name = get_bucket_and_path_from(full_bucket_path)
+        try:
+            file_size = self.repository.objects.download_to_file(
```

Comment:
nit: the `download_to_file()` method seems a bit confusing with the `download_file()`.  Perhaps, tweak `download_file()` to just `download()`?

Reply:
Done.